deploy/addons/assets.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import "embed"
+
+var (
+	// AutoPauseAssets assets for auto-pause addon
+	//go:embed auto-pause/*.tmpl
+	//go:embed auto-pause/auto-pause.service
+	//go:embed auto-pause/unpause.lua
+	AutoPauseAssets embed.FS
```

Comment:
why this vars are not used anywhere?

Reply:
They are used in `pkg/minikube/assets/addons.go`.